### PR TITLE
Integrate kylas engine

### PR DESCRIPTION
**Pull request by aditya-selldo**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -6,6 +6,8 @@ ruby "3.1.0"
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
 gem "rails", "~> 7.0.4"
 
+gem 'kylas_engine', git: 'https://github.com/amuratech/kylas_engine.git'
+
```
Comment (sarang-selldo) [inline Gemfile:10]: Please check kylas engine gemspec file and remove gem which are present in engine already.

File: config/application.rb [hunk 2]
```diff
@@ -12,7 +12,8 @@
 # require "action_text/engine"
 require "action_view/railtie"
 # require "action_cable/engine"
-
+# require "rails/test_unit/railtie"
```
Comment (sarang-selldo) [inline config/application.rb:15]: remove this commented line

File: app/assets/config/manifest.js [hunk 3]
```diff
@@ -1,2 +1,8 @@
 //= link_tree ../images
 //= link_directory ../stylesheets .css
+//= link_directory ../stylesheets .scss
+//= link_directory ../javascripts .js
+// link kylas_engine_manifest.js
+//= link kylas_engine/logo.png
```
Comment (sarang-selldo) [inline app/assets/config/manifest.js:6]: Please remove for all images

**Review by sarang-selldo — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (aditya-selldo) [reply Gemfile:10]: Done

File: config/application.rb (hunk 2 above)
Comment (aditya-selldo) [reply config/application.rb]: Done

File: app/assets/config/manifest.js (hunk 3 above)
Comment (aditya-selldo) [reply app/assets/config/manifest.js]: Done

**Review by sarang-selldo — APPROVED** [review]

**Pull request merged**